d2/src/main/java/com/linkedin/d2/jmx/RelativeLoadBalancerStrategyJmx.java

```diff
@@ -0,0 +1,168 @@
+package com.linkedin.d2.jmx;
+
+import com.linkedin.d2.balancer.clients.TrackerClient;
+import com.linkedin.d2.balancer.strategies.LoadBalancerQuarantine;
+import com.linkedin.d2.balancer.strategies.relative.RelativeLoadBalancerStrategy;
+import com.linkedin.d2.balancer.strategies.relative.TrackerClientState;
+import com.linkedin.d2.balancer.util.partitions.DefaultPartitionAccessor;
+import com.linkedin.util.degrader.CallTracker;
+import java.net.URI;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+
+public class RelativeLoadBalancerStrategyJmx implements RelativeLoadBalancerStrategyJmxMBean
+{
+  private final RelativeLoadBalancerStrategy _strategy;
+
+  public RelativeLoadBalancerStrategyJmx(RelativeLoadBalancerStrategy strategy)
+  {
+    _strategy = strategy;
+  }
+
+  @Override
+  public double getLatencyStandardDeviation()
+  {
+    Map<TrackerClient, TrackerClientState> stateMap =
+        _strategy.getPartitionState(DefaultPartitionAccessor.DEFAULT_PARTITION_ID).getTrackerClientStateMap();
+
+    List<Double> weightedLatencies = stateMap.keySet().stream()
+        .map(trackerClient -> getWeightedLatency(trackerClient.getLatestCallStats()))
+        .collect(Collectors.toList());
+
+    double avgLatency = weightedLatencies.stream()
+        .mapToDouble(Double::doubleValue)
+        .average()
```

Comment:
Is it possible that a host does not get any traffic so the latency is 0, will it be counted here?

Reply:
Can we add some unit test on this to cover some different cases?

Reply:
> Is it possible that a host does not get any traffic so the latency is 0, will it be counted here?

Since I had the `orElse(0)` clause, it shouldn't be a problem it a host doesn't get any traffic.

Reply:
> Can we add some unit test on this to cover some different cases?

Sure, I've written some local unit tests. Would you like me to also submit those as official code changes?

Reply:
I mean for the host that does not have any traffic, we probably don't want to count it in the deviation. For example if we have 5 hosts in total, for some reason 4 of them have avg host latency 30, 32, 32, 34 and the other one does not have any traffic, hence the host latency is 0. When we calculate the deviation, I'm thinking about if we should only use 30, 32, 32, 34 as data points.

Reply:
Yes it would be great to always have some unit test coverage

Reply:
Unit tests added in the new commit.